lib/vue/src/components/resourceList/CResourceList.vue

```diff
@@ -126,65 +126,72 @@
     </b-card-body>
 
     <template
-      v-if="showFooter"
       #footer
     >
-      <div class="d-flex gap-1 align-items-center">
-        <div
-          v-if="!hideTotal"
-          class="text-nowrap"
-        >
-          {{ getPagination }}
-        </div>
+      <div
+        class="d-flex align-items-center flex-wrap justify-content-between w-100"
+      >
+        <div class="d-flex gap-col-3 align-items-center flex-wrap">
+          <div
+            v-if="!hideTotal"
+            class="text-nowrap"
+          >
+            {{ getPagination }}
+          </div>
         
-        <div>
-          <b-form-select
-            v-model="pagination.limit"
-            :options="perPageOptions"
-            @change="loadRows"
-          />
+          <div class="d-flex flex-wrap gap-1 align-items-center">
+            <div>
+                {{ $t('general:resourceList.pagination.recordsPerPage') }}
+            </div>
+
+            <b-form-select
+              :options="perPageOptions"
+              :value="pagination.limit"
```

Comment:
:value
:options

value is always priority, same reason v-model is always first